pkg/controller/storagecluster/cephcluster.go

```diff
@@ -51,7 +51,7 @@ func (r *ReconcileStorageCluster) ensureCephCluster(sc *ocsv1.StorageCluster, re
 		}
 	}
 
-	if sc.Spec.Network.IsMultus() {
+	if isMultus(sc.Spec.Network) {
```

Comment:
Don't we need to add NetworkSpec to CephCluster if it isMultus?

Reply:
`ensureCephCluster()` returns cephCluster spec only so I didn't put it inside `isMultus()` in order not to change the function definition.